spec/unit/yardstick/document/path_spec.rb

```diff
@@ -5,8 +5,8 @@
 describe Yardstick::Document, '#path' do
   subject { described_class.new(docstring).path }
 
-  let(:docstring) { double('docstring', :object => object) }
-  let(:object)    { double('object', :path => 'Foo#bar')   }
+  let(:docstring) { double('docstring', object: object) }
+  let(:object)    { double('object', path: 'Foo#bar')   }
 
   it { should eq('Foo#bar') }
```

Comment:
Not that it is required for this PR, given that this exists in the source and there was one more `eq` -> `eql` found, I wonder if we should just do a find/replace of all `'eq('` strings into `'eql('`?